src/components/Contacts/ContactRow.tsx

```diff
@@ -37,55 +36,35 @@ interface Props {
 
 export const ContactRow: React.FC<Props> = ({ contact }) => {
   const classes = useStyles();
-  function isLate(): GiftStatusEnum {
-    const date = new Date();
 
-    if (contact.lateAt == null) {
-      return GiftStatusEnum.Hidden;
-    }
+  const contactHasBirthday = (): boolean =>
+    contact.people.nodes.some((person) =>
+      Interval.after(DateTime.fromJSDate(new Date()).startOf('day'), {
+        days: 5,
+      }).contains(
+        DateTime.fromISO(
+          `${new Date().getFullYear}-${person.birthdayMonth}-${
+            person.birthdayDay
+          }`,
+        ),
```

Comment:
Luxon has `DateTime.fromObject(...)` for this. Took me a bit to find. Also I missed that month and day were separate fields (I still had an ISO string I was playing with in an example) and than the year needs to be ignored in the comparison. Thanks for fixing my last suggestion to account for those :)
```suggestion
      DateTime.fromObject({ month: person.birthdayMonth, day: person.birthdayDay }),
```